GispoCoding/aukigo-backend
# Follow TileJSON spec and various improvements

Follow TileJSON spec and various imporvements:
* Wait for db during tests
* Use REST api for models
* Load OSM data as scheduled task
* Delete features if they are not included in responses
* Add models for WMTS and vector tile basemaps

Issues: Fixes #20, fixes #15, fixes #18  and fixes #21.


<details>
  <summary>Testing instructions</summary>

1. `docker-compose up -d --build`
2. Load test fixture with: `docker-compose exec web python manage.py loaddata camping.json basemaps.json`
3. Populate data with by going to http://localhost:8001/api/populate_osm/
4. After few seconds go to http://localhost:8001/api/ and browse trough grawlable api
5. Check the capabilities document at http://localhost:8001/api/capabilities/



</details>
